### ISSUE TITLE
Replace anatomic by anatomical in terminologies

### ISSUE THREAD

**Issue by lassoan** [Slicer/Slicer#5689]
It would be more appropriate to consistently use "anatomical" word instead of "anatomic" when talking about anatomical regions (related to terminologies). When it is possible to rename terminology contexts (https://github.com/Slicer/Slicer/issues/5644 is fixed) then "anatomic" should be renamed to "anatomical" throughout the code (terminology context name, GUI, API, source code, and comments) for consistency. When changing the API, the new "...Anatomical..." method names should be added and the existing "...Anatomic..." methods should be deprecated.

## Environment
- Slicer version: Slicer-4.13.0-2021-06-11
- Operating system: all

**Comment by pieper** [comment]
I agree with @jamesobutler that 'anatomical' is more common in English usage, but 'anatomic' doesn't sound that bad to me.  I think it's like 'surgery' vs. 'surgical' or where the usages get mixed sometimes as [people say here](https://english.stackexchange.com/questions/161665/is-there-a-difference-between-anatomic-and-anatomical).  [But this page](https://wikidiff.com/anatomic/anatomical) claims to define the difference, which would give the nod to 'anatomic' when taking about structures and 'anatomical' about observations.  I think that we should for sure be consistent and that we should defer to the dicom usage.

**Comment by lassoan** [comment]
DICOM standard is unlikely to change, so consistency with that sounds like a good idea. DICOM uses anatomic - http://dicom.nema.org/medical/dicom/current/output/chtml/part16/sect_CID_4.html

@pieper do you think all usage of word "anatomical" could be changed to "anatomic" on this page: https://slicer.readthedocs.io/en/latest/user_guide/modules/terminologies.html ? If yes, then I would change that and would close this anatomic/anatomical debate once and for all (using anatomic everywhere in relation to terminologies).

**Comment by cpinter** [comment]
Thanks @pieper for the objective research on this topic. I guess we foreigners just accepted the judgement of a native there. According to the structure vs observation difference, we are much more on the structure side then the observation side, so that's one point for anatomic. Also agree that sticking to DICOM conventions is important and if DICOM uses anatomic, then we also should (2:0). Probably it would then be better to switch back to "anatomic".

@jamesobutler do you agree after seeing these findings?

**Comment by jamesobutler** [comment]
Unfortunately DICOM doesn't clearly use "anatomic" over "anatomical" as if you search the table of contents http://dicom.nema.org/medical/dicom/current/output/chtml/part16/PS3.16.html they use "Anatomical Structures for Radiotherapy", but then use "Anatomic Structures".  

I would say in general the more common usage would be "Anatomical Region" as @pieper has suggested, while "Anatomic Region" may still be used it is just not the preferred term in English or at least American English.  With DICOM being an international standard, it has likely been influenced by non-native English speakers.

I think in writing it would be acceptable if you said something like "The terminology context codes are a subset from the ["Anatomic Region Codes"](http://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_L.html) of the DICOM standard".  In that sentence you are clearly defining that you are referring to how the DICOM standard calls it and that makes it easier to use appropriate search terms when looking up the standard.

**Comment by pieper** [comment]
I've never asked, but I suspect that David Clunie would lean toward British English and he might have let uses of Anatomical slip through because he didn't want o make a fuss given the ambiguities of American English.  From a quick skim of the standard it looks like more modern parts (e.g. SEG) use Anatomic and older parts (e.g. RT) use Anatomical.  

In any case it sounds like we're all in agreement that we want to be (as much as possible) faithful to the standard and consistent between code and documentation.
